PWGCF/FemtoWorld/Core/FemtoWorldParticleHisto.h

```diff
@@ -14,11 +14,12 @@
 /// \author Andi Mathis, TU München, andreas.mathis@ph.tum.de
 /// \author Zuzanna Chochulska, WUT Warsaw, zchochul@cern.ch
 
-#ifndef FEMTOWORLDPARTICLEHISTO_H_
-#define FEMTOWORLDPARTICLEHISTO_H_
+#ifndef PWGCF_FEMTOWORLD_CORE_FEMTOWORLDPARTICLEHISTO_H_
+#define PWGCF_FEMTOWORLD_CORE_FEMTOWORLDPARTICLEHISTO_H_
 
 #include "PWGCF/FemtoWorld/DataModel/FemtoWorldDerived.h"
 #include "Framework/HistogramRegistry.h"
+#include <string>
```

Comment:
MegaLinter requires you to put C style headers, i.e <...>, before any other so put them above line 22